src/app/components/wrapper-components/rules-configuration/rules-configuration.component.html

```diff
@@ -9,15 +9,20 @@
                 <select id="ruleSelect"
                         [disabled]="editable === false"
                         (change)="onChange($event)">
-                    <option [value]="rulesConfigDescription.getDefaultConfig().name()"
-                            selected>
+                    <option id="config-dropdown-{{ rulesConfigDescription.getDefaultConfig().name() }}"
```

Comment:
Ceci est une ancre pour un commentaire plus loin dans la review : "ici"

Reply:
id are good !

Reply:
nor for option selections! "<option id="id-{{option.name()}}">{{ option.name() }}</option>" is redundancy at its finest

Reply:
Non, non, non !
C'est très bien !

Reply:
nope, plz remove useless ids!

Reply:
ids are never useless, except if they are not unique !